seatunnel-examples/seatunnel-flink-connector-v2-example/src/main/java/org/apache/seatunnel/example/flink/v2/SeaTunnelApiExample.java

```diff
@@ -31,7 +31,7 @@
 public class SeaTunnelApiExample {
 
     public static void main(String[] args) throws FileNotFoundException, URISyntaxException, CommandException {
-        String configFile = getTestConfigFile("/examples/fake_to_console.conf");
+        String configFile = getTestConfigFile("/examples/fake_to_dingtalk.conf");
```

Comment:
Please don't change this default config. We should make sure other developer can run this example easy. You can create a SeaTunnelDingTaskExample.java or create a e2e test for DingTalk.

Reply:
done